tools/manta/manta.xml

```diff
@@ -1,15 +1,12 @@
-﻿<tool id="manta" name="Manta" version="@WRAPPER_VERSION@">
-
+<tool id="manta" name="Manta" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.05">
     <description>Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads.</description>
-
     <macros>
         <import>manta_macros.xml</import>
     </macros>
     <expand macro="requirements"/>
     <expand macro="stdio"/>
 
     <command detect_errors="exit_code"><![CDATA[
-    @VERSION@
```

Comment:
was that needed @drosofff ?

Reply:
I don't think so ! I am ok with removing it